src/main/proto/ess.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+option java_multiple_files = true;
+package org.ess;
+
+service EncryptedStorageService {
+  rpc get(GetRequest) returns (GetResponse);
+  rpc put(PutRequest) returns (PutResponse);
+  //TODO: Add getKeysSummary (Gives version numbers for all keys in a store) and list by prefix API's
+}
```

Comment:
What is being "get" or "put"? I'd recommend following these naming conventions for both method and type names:

https://cloud.google.com/apis/design/naming_convention

Reply:
Good point, changed it to GetObjectRequest and PutObjectRequest